Hub-Server/src/main/resources/application.properties

```diff
@@ -4,3 +4,5 @@ quarkus.datasource.db-kind=h2
 quarkus.datasource.jdbc.url=jdbc:h2:./data/database
```

Comment:
did you want to make this variable based on the env variable?

Reply:
Well, this is in hub so we expect only one instance to ever be ran, so we just left the name generic. It's easy to change, for consistency should we change it to be an env variable in hub as well?

Reply:
i think for consistency, probbaly a good idea.